signac/diff.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import signac
```

Comment:
Use relative imports. Avoid absolute imports like `import signac` inside the `signac` package directory. Something like this would work:
```python
from .contrib.collection import _traverse_filter
```